### ISSUE TITLE
Feature: automatically enable Push Notifications entitlement in iOS

### ISSUE THREAD

**Issue by jpduckwo** [fechanique/cordova-plugin-fcm#140]
It would be great if the plugin could add the aps-environment capability to the project automatically. Currently you need to enable this manually when you create an iOS platform.

It is stored in the *.entitlements file for the project as 
```
<dict>
	<key>aps-environment</key>
	<string>development</string>
</dict>
```

![image](https://cloud.githubusercontent.com/assets/13150911/19298205/8774c880-9097-11e6-9790-8c32877e602d.png)

Another setting is Background Mode, Remote notifications which is stored in the *-Info.plist file as 
```
    <key>UIBackgroundModes</key>
    <array>
      <string>remote-notification</string>
    </array>
```

![image](https://cloud.githubusercontent.com/assets/13150911/19298269/fac51bf0-9097-11e6-8fda-fe62d990f431.png)

**Comment by alizahid** [comment]
When I installed this plugin, push notifications worked without any extra configuration on Android, but not on iOS. Adding this config fixes that.

**Comment by msyahidin** [comment]
+1 for this issue

**Comment by findel** [comment]
Yes I had this issue too. It's taken AGES to figure out what was needed. This should be done automatically if possible.

**Comment by rwoody** [comment]
@findel What did you do?

I'm not using this plugin but `cordova-plugin-firebase` rather. Another plugin `cordova-universal-links-plugin` was overwriting at least one entitlements file (debug), and my entitlements file therefore only had the entitlements for associated domains.

See also:
nordnet/cordova-universal-links-plugin/issues/89
nordnet/cordova-universal-links-plugin#69
